honeybee_radiance/command/options/oconv.py

```diff
@@ -0,0 +1,115 @@
+"""Oconv parameters."""
+from .optionbase import OptionCollection, FileOption, TupleOption, IntegerOption, \
+    BoolOption
+
+
+class OconvOptions(OptionCollection):
+    """
+    [ -i octree | -b xmin ymin zmin size ][ -n objlim ][ -r maxres ][ -f ][ -w ]
+
+    Also see: https://floyd.lbl.gov/radiance/man_html/oconv.1.html
+    """
+
+    __slots__ = ('_i', '_b', '_n', '_r', '_f', '_w')
+
+    def __init__(self):
+        """oconv command options."""
+        OptionCollection.__init__(self)
+        self._i = FileOption('i', 'existing octree file')
+        self._b = TupleOption(
+            'b', 'bounding box as min_x, min_y, min_z and size', None, 4, float
+        )
+        self._n = IntegerOption(
+            'n', 'maximum surface set size for each voxel - default: 6', min_value=0
+        )
+        self._r = IntegerOption(
+            'r', 'maximum octree resolution - default: 16384', min_value=0
+        )
+        self._f = BoolOption('f', 'frozen octree - default: off')
+        self._w = BoolOption('w', 'warning messages - default: on')
+        self._on_setattr_check = True
+
+    def on_setattr(self):
+        assert not (self.b.is_set and self.i.is_set), \
+            'The -b and -i options are mutually exclusive.'
+
+    @property
+    def i(self):
+        """input octree."""
+        return self._i
+
+    @i.setter
+    def i(self, value):
+        self._i.value = value
+
+    @property
+    def b(self):
+        """Scene bounding cube.
+        
+        The -b option allows the user to give a bounding cube for the scene, starting at
+        xmin ymin zmin and having a side length size. If the cube does not contain all of
+        the surfaces, an error results.
+        
+        The -b and -i options are mutually exclusive.
+        """
+        return self._b
+
+    @b.setter
+    def b(self, value):
+        self._b.value = value
+
+    @property
+    def n(self):
+        """maximum surface set size for each voxel.
```

Comment:
For consistency sake with documentation, you should probably capitalize Maximum.  The same goes for a bunch of other docstrings on this object.

Reply:
And for nearly all of the docstrings copy/pasted from radiance for these parameters.